packages/api/dev.export.sh

```diff
@@ -7,6 +7,10 @@ cp ../webapp/public/locales/en/crop.json src/jobs/locales/en
 cp ../webapp/public/locales/es/crop.json src/jobs/locales/es
 cp ../webapp/public/locales/pt/crop.json src/jobs/locales/pt
 cp ../webapp/public/locales/fr/crop.json src/jobs/locales/fr
+cp ../webapp/public/locales/fr/crop.json src/jobs/locales/de
```

Comment:
Tempted to just copy the whole locales folder just so we don't have to keep adding new lines?

Reply:
You mean copying the whole webapp `locales/` folder? No way it's huge! `crop.json` is not nearly the largest of the files. But I'm pretty sure there is a better way to write this more efficiently.

Reply:
Lol theres got to be a glob or something! haha.. anyways would be nice to have one less thing in the future!

Reply:
I totally missed this bug I pointed out before haha woops!

Reply:
I didn't even realize you were talking about a bug! I thought you were just saying the line by line copying is inefficient!

Reply:
I was saying that haha, so I didnt either! haha... but I did point it out by accident despite my not noticing the real problem haha